### PR TITLE
Dev wwp

### PR DESCRIPTION
**Pull request by gouws-j**
[code_review_checklist_14_60.md](https://github.com/witseie-elen4010/2022-group-lab-011/files/8760227/code_review_checklist_14_60.md)

### CODE REVIEW

**Comment by gouws-j** [comment]
[pull_request_checklist_14_60.md](https://github.com/witseie-elen4010/2022-group-lab-011/files/8760278/pull_request_checklist_14_60.md)

File: package.json [hunk 1]
```diff
@@ -14,7 +14,11 @@
...
   "homepage": "https://github.com/witseie-elen4010/2022-group-lab-011#readme",
   "dependencies": {
-    "ejs": "3.1.8",
-    "express": "4.18.1"
+    "ejs": "^3.1.8",
+    "express": "^4.18.1",
+    "mongoose": "^6.3.4"
+  },
+  "devDependencies": {
+    "nodemon": "^2.0.16"
```
Comment (robinjonker) [inline package.json:22]: Please remove all devDependencies, and mongoose is related to setting up the database for MongoDB if I am right? Why is that needed for these changes? If not, please don't make alterations in package.son or package-lock.json

**Review by robinjonker — CHANGES_REQUESTED** [review]

File: public/main.js [hunk 2]
```diff
@@ -1,13 +1,50 @@
...
+  let availableSpace = 1  
   let numGuessCount = 0
 
+  /* the word might be very obscure so the console out of
+  the word can be used to test
+  
+  the 5000 delay is to give enough time for the getNewWord() function to
+  interact with the API*/
+  
+  setTimeout(() => { console.log(`${word}`); }, 5000)
```
[the remark this answers is not in the document]
Comment (gouws-j) [reply public/main.js:17]: good idea, i was manually changing the amounts -_-

File: package.json (hunk 1 above)
Comment (gouws-j) [reply package.json:22]: I was setting up mongoose for the potential implementation of a database to become the dictionary. I will remove for now.

Comment (robinjonker) [reply package.json:22]: Okay perfect, in that case please remove all changes within both .json files

File: public/main.js (hunk 2 above)
[the remark this answers is not in the document]
Comment (gouws-j) [reply public/main.js:17]: I tried using the async await setup. The getNewWord() takes 1 millsec to complete, but the word takes 4 seconds to update. I'll have to look into hows the send and rec works in depth before I can do this implementation. Can I look into next sprint?

[the remark this answers is not in the document]
Comment (robinjonker) [reply public/main.js]: That's perfect, create a new issue for it and it can be done next sprint

**Review by robinjonker — APPROVED** [review]
[Code_Review_62.md](https://github.com/witseie-elen4010/2022-group-lab-011/files/8761208/Code_Review_62.md)
